docker-images/kafka-based/kafka/scripts/kafka_run.sh

```diff
@@ -116,6 +116,10 @@ export KAFKA_OPTS
 # Configure Garbage Collection logging
 . ./set_kafka_gc_options.sh
 
+if [ -f "/tmp/kafka_server_sasl_plain_jaas.conf" ]; then
+  KAFKA_OPTS="${KAFKA_OPTS} -Djava.security.auth.login.config=/tmp/kafka_server_sasl_plain_jaas.conf"
```

Comment:
recomend to do some validations on file and print errors
simple validations like size(non empty file) can be done